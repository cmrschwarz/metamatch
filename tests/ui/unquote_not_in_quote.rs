use metamatch::unquote;

fn main() {
    let res = unquote! {
        [<unquote>]
        42
        [</unquote>]
    };

    assert_eq!(res, 10);
}
