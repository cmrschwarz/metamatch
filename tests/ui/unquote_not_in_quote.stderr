error: template tags besides `quote` are only supported in quoted output
 --> tests/ui/unquote_not_in_quote.rs:5:9
  |
5 |         [<unquote>]
  |         ^^^^^^^^^^^
